neo/Prompt/Commands/Wallet.py

```diff
@@ -41,11 +47,19 @@ def execute(self, arguments):
         wallet = PromptData.Wallet
         item = get_arg(arguments)
 
-        # Create and Open must be handled specially.
+        # Create, Open, and Close must be handled specially.
         if item == 'create':
             self.execute_sub_command(item, arguments[1:])
             return
 
+        elif item == 'open':
```

Comment:
We could regroup all the commands in a `set` and check if `item in xxx`. (if `close` can be moved out, see after)